website/project/model.py

```diff
@@ -4584,3 +4621,28 @@ def approve(self, user):
     def reject(self, user):
         self.approval.reject(user)
         self.approval.save()
+
+
+@unique_on(['destination_node', '_id'])
+class MailingListEventLog(StoredObject):
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
```

Comment:
We're trying to keep `project.model` from growing any larger than it already is. Can you put this model and other mailing-list-related code in its own module, e.g. `website.mailing_lists`?